src/redux/covid/covid.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable consistent-return */
```

Comment:
- [ ] Linting is important to reduce errors and improve the overall quality of your code  so I think fixing them would be much better than disabling them